hswindow/hstoolwindow/hsprojectexplorer/projectexplorer.go

```diff
@@ -138,13 +138,21 @@ func (m *ProjectExplorer) GetProjectTreeNodes() g.Layout {
 		return []g.Widget{g.Label("No project loaded...")}
 	}
 
-	fileStructure := m.project.GetFileStructure()
+	fileStructure, err := m.project.GetFileStructure()
+	if err != nil {
+		dialog.Message(err.Error()).Error()
+	}
 
 	if fileStructure == nil {
 		return []g.Widget{g.Label("No file structure detected...")}
 	}
 
-	return []g.Widget{m.renderNodes(m.project.GetFileStructure())}
+	nodes, err := m.project.GetFileStructure()
+	if err != nil {
+		return []g.Widget{g.Label(err.Error())}
+	}
+
+	return []g.Widget{m.renderNodes(nodes)}
```

Comment:
why not `g.Layout(m.renderNodes(nodes))`?

Reply:
good point. I think it's just old code, I kept the style. updating...